modules/rosapps/applications/cmdutils/arping/arping.c

```diff
@@ -107,6 +107,8 @@ void FormatOutput(UINT uID, ...)
 
     if(pBuf != Buf)
         LocalFree(pBuf);
+
+    va_end(valist);
```

Comment:
This actually doesn't fix this function's problem. You can't use a va_list twice as is done here.
Can you change it so it calls va_end after the first function call, then does another va_start/va_end pair around the second? Thanks.

Reply:
I'll do that.
This last `va_end()` could be done much sooner too, couldn't it? Or must `valist` remain active until then?

Reply:
Correct, you can do va_start(); function(); va_end(); both times.

Reply:
Done.